.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: python
```

Comment:
Curious about `.travis.yml`, does it actually work? Like is travis running CI for our fork? Or is this a carryover from upstream?

If it works I think we should add Python 3.7 and default to it. It not we can remove it. Edit: Looks like no CI so we can likely just remove this file.

Reply:
Looks like we're pointing at the wrong url for travis.  Some of our old commits have travis builds, but it looks like we need to switch from travis-ci.org to travis-ci.com

https://travis-ci.org/github/discord/redis-py/pull_requests

```
Since June 15th, 2021, the building on travis-ci.org is ceased. Please use travis-ci.com from now on.
```